tensorflow/inference/docker/2.8/py3/cu112/Dockerfile.gpu

```diff
@@ -165,3 +174,65 @@ RUN curl https://aws-dlc-licenses.s3.amazonaws.com/tensorflow-2.7/license.txt -o
 RUN rm -rf /tmp/*
 
 CMD ["/usr/bin/tf_serving_entrypoint.sh"]
+
```

Comment:
@hsl89 - can the commented lines regarding TensorRT in this file be removed?

Reply:
yes, they are from the previous release